assets/terraform-modules/packet/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -11,6 +11,7 @@ module "bootkube" {
   api_servers_external = [format("%s.%s", var.cluster_name, var.dns_zone)]
   etcd_servers         = [for i, d in packet_device.controllers : format("%s-etcd%d.%s", var.cluster_name, i, var.dns_zone)]
   asset_dir            = var.asset_dir
+  api_key              = var.api_key
```

Comment:
Can you elaborate in the commit as to what it does?

Reply:
Cool